src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs

```diff
@@ -1039,7 +1039,7 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                 preprocessorSymbols: defines.ToImmutableAndFree(),
                 documentationMode: parseDocumentationComments ? DocumentationMode.Diagnose : DocumentationMode.None,
                 kind: SourceCodeKind.Regular
-            );
+            ).WithFeatures(features.ToImmutableDictionary(feature => feature, feature => "true"));
```

Comment:
We should add an overload, even if it's only internal, which takes features as a parameter.  As written we're unnecessarily allocating `CSharpParseOption` instances